### PR TITLE
MODLD-333: Code cleanup - remove parentPredicate from configuration

### PR DESCRIPTION
**Pull request by pkjacob**

### CODE REVIEW

File: src/main/java/org/folio/marc4ld/service/marc2ld/Marc2BibframeMapperImpl.java [hunk 1]
```diff
@@ -160,4 +138,32 @@ private void cleanEmptyEdges(Resource resource) {
...
   }
+
+  private InstanceAndWork createInstanceAndWork() {
+    var work = new Resource().addType(WORK);
+    var instance = new Resource().addType(INSTANCE);
+    instance.getOutgoingEdges().add(new ResourceEdge(instance, work, INSTANTIATES));
+    return new InstanceAndWork(instance, work);
+  }
+
+  private void setLabelsAndId(InstanceAndWork instanceAndWork) {
```
Comment (askhat-abishev) [inline src/main/java/org/folio/marc4ld/service/marc2ld/Marc2BibframeMapperImpl.java:149]: setLabelsAndIds

File: src/main/java/org/folio/marc4ld/service/marc2ld/Marc2BibframeMapperImpl.java [hunk 2]
```diff
@@ -160,4 +138,32 @@ private void cleanEmptyEdges(Resource resource) {
...
+    setLabelAndId(instanceAndWork.instance);
+    setLabelAndId(instanceAndWork.work);
+  }
+
+  private void setLabelAndId(Resource resource) {
+    resource.setLabel(selectLabelFromTitle(resource));
+    resource.setId(hashService.hash(resource));
+  }
+
+  private String selectLabelFromTitle(Resource instance) {
```
Comment (askhat-abishev) [inline src/main/java/org/folio/marc4ld/service/marc2ld/Marc2BibframeMapperImpl.java:159]: private String selectLabelFromTitle(Resource **resource**) {

**Review by askhat-abishev — APPROVED** [review]
we can rename EMPTY constant to SPACE in this PR

File: src/main/java/org/folio/marc4ld/service/marc2ld/Marc2BibframeMapperImpl.java (hunk 1 above)
Comment (aleksei-pronichev) [inline src/main/java/org/folio/marc4ld/service/marc2ld/Marc2BibframeMapperImpl.java:149]: what about encapsulating a bit
like `setAdditionInfo()`

**Review by aleksei-pronichev — APPROVED** [review]

File: src/main/java/org/folio/marc4ld/service/marc2ld/Marc2BibframeMapperImpl.java (hunk 1 above)
Comment (pkjacob) [reply src/main/java/org/folio/marc4ld/service/marc2ld/Marc2BibframeMapperImpl.java]: renamed to setAdditionalInfo

**Pull request merged**